pkg/vfs/vfs_unix.go

```diff
@@ -179,13 +182,15 @@ func (v *VFS) SetAttr(ctx Context, ino Ino, set int, opened uint8, mode, uid, gi
 	if set&meta.SetAttrMtime != 0 {
 		attr.Mtime = mtime
 		attr.Mtimensec = mtimensec
-		v.writer.UpdateMtime(ino, time.Unix(mtime, int64(mtimensec)))
-	}
-	if set&meta.SetAttrMtimeNow != 0 {
-		v.writer.UpdateMtime(ino, time.Now())
 	}
 	err = v.Meta.SetAttr(ctx, ino, uint16(set), 0, attr)
```

Comment:
When m.Write() is called between SetAttr() and UpdateMtime(), then mtime will be wrong.

Reply:
should we always update `mtime` even if the `setattr` is denied?

Reply:
We could check permission first, then call UpdateMtime before SetAttr

Reply:
done

Reply:
done